Reneiski_Homework1/ReneiskiSelenium11/Tests/Tests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using NUnit.Framework;
+using ReneiskiSelenium11.PageObjects.ButtonPage;
+using ReneiskiSelenium11.PageObjects.CheckBoxPage;
+using ReneiskiSelenium11.PageObjects.LinksPage;
+using ReneiskiSelenium11.PageObjects.RadioButtonPage;
+using ReneiskiSelenium11.PageObjects.WebTablePage;
+using ReneiskiSelenium11.PageObjects.WebTablePage.RegistrationForm;
+
+namespace ReneiskiSelenium11.Tests
+{
+    public class Selenium : BaseTest
+    {
+
+        [Test]
+        public void FirstCheckBoxTest()
+        {
+            CheckBoxPage check = new CheckBoxPage(_driver);
+
+            check.GoToPage("checkbox");
+
+            check.ClickCheckBox();
+
+            Assert.IsTrue(check.GetAttributeCheckBox());
+
+            Assert.AreEqual(check.ExpectedText(), check.TextResult());
+        }
+
+        [Test]
+        public void SecondRadioButtonTest()
+        {
+            RadioButtonPage buttonPage = new RadioButtonPage(_driver);
+
+            buttonPage.GoToPage("radio-button");
+
+            buttonPage.yesRadiobtn.Click();
+
+            Assert.AreEqual("You have selected Yes", buttonPage.TextYesResult());
+
+            buttonPage.impressiveRadioBtn.Click();
+
+            Assert.AreEqual("You have selected Impressive", buttonPage.TextYesResult());
+        }
+
+        [Test]
+        public void ThirdWebTablesTest()
+        {
+            WebTablePage webTablePage = new WebTablePage(_driver);
+            RegFormPage regForm = new RegFormPage(_driver);
+
+            webTablePage.GoToPage("webtables");
+
+            Assert.AreEqual("Cierra", webTablePage.ReturnText(webTablePage.cCierra));
+            Assert.AreEqual("Vega", webTablePage.ReturnText(webTablePage.cVega));
+            Assert.AreEqual("39", webTablePage.ReturnText(webTablePage.cAge));
+            Assert.AreEqual("cierra@example.com", webTablePage.ReturnText(webTablePage.cEmail));
+            Assert.AreEqual("10000", webTablePage.ReturnText(webTablePage.cSalary));
+            Assert.AreEqual("Insurance", webTablePage.ReturnText(webTablePage.cDepartament));
+
+            webTablePage.ClickBtn("addNewRecordButton");
+
+            regForm.FillInValues("firstName");
+            regForm.FillInValues("lastName");
+            regForm.FillInValues("userEmail");
+            regForm.FillInValues("age");
+            regForm.FillInValues("salary");
+            regForm.FillInValues("department");
+
+            webTablePage.ClickBtn("submit");
+
+            Assert.AreEqual($"{regForm.firstName}\r\n{regForm.lastName}\r\n{regForm.age}\r\n{regForm.userEmail}\r\n{regForm.salary}\r\n{regForm.department}", webTablePage.ReturnTextResult());
```

Comment:
it's better to format actual string and replace all r\n\ with " ", and write expected string with rn

Reply:
Fixed